access-token-management/src/AccessTokenManagement/OTel/AccessTokenManagementMetrics.cs

```diff
@@ -7,7 +7,6 @@
 namespace Duende.AccessTokenManagement.OTel;
 
 
-[Obsolete(Constants.AtmPublicSurfaceInternal, UrlFormat = Constants.AtmPublicSurfaceLink)]
```

Comment:
I didn't expect to see things getting taken off the Obsolete list.

Reply:
After consideration, this one should not have been internal.